languages/CookieConsent/en.ini

```diff
@@ -0,0 +1,42 @@
+Accept All Cookies = "Accept All Cookies"
+Accept Only Essential Cookies = "Accept Only Essential Cookies"
+allow_video_always = "Show Video and Always Allow"
+allow_video_once = "Show Video"
+analytics_cookies_description_html = "Analytics collected with these cookies is used for development of the site."
+analytics_cookies_title_html = "Analytics Cookies"
+category_description_html = "Cookies used on the site are categorized based on their purpose. See below for description of each category as well as the option to accept or reject it. <a href="Content/privacy" target="_blank">Privacy information for the site</a>."
+cookie_description_cart_html = "Used to store the contents of the book bag."
+cookie_description_consent_html = "Used to store information about whether visitors have given consent or declined the use of cookie categories used on the site."
+cookie_description_language_html = "Used to store the current user interface language."
+cookie_description_matomo_id_html = "Used for website statistics. Placed by Matomo to store a unique user ID."
+cookie_description_matomo_ref_html = "Used for website statistics. Placed by Matomo to store referrer ID's."
+cookie_description_matomo_ses_html = "Used for website statistics. Placed by Matomo to store a unique session ID."
+cookie_description_search_open_html = "Used to store open records in search results."
+cookie_description_session_html = "Used to maintain user's session state."
+cookie_description_sidefacet_html = "Used to maintain facet expand/collapse state."
+cookie_description_ui_html = "Used to store the current user interface."
+cookie_description_youtube_bandwidth_html = "Used to estimate bandwidth."
+cookie_description_youtube_gps_html = "Used to store user's location."
+cookie_description_youtube_marketing_html = "Used to track user's actions."
+cookie_description_youtube_preferences_html = "Used to store user's preferences."
```

Comment:
See below for some explanation. Not important to us, though, since we'll just add it locally as necessary, but could be useful as an example.

Reply:
Since the YouTube strings don't actually mention YouTube in any way, is there a way to make the strings more generic/reusable so that other similar services can leverage them without the need to customize language files?